app/src/main/scala/org/alephium/explorer/persistence/queries/TransactionQueries.scala

```diff
@@ -351,24 +357,25 @@ object TransactionQueries extends StrictLogging {
         .sortBy(_.outputOrder)
         .map(_.toApiOutput())
     }
-    val gasByTx = gases.groupBy(_.txHash).view.mapValues(_.map(_.gasInfo()))
+    val gasByTx = gases.groupBy(_.txHash).view.mapValues(_.map(_.info()))
     txHashesTs.map { txn =>
-      val ins                   = insByTx.getOrElse(txn.txHash, ArraySeq.empty)
-      val ous                   = ousByTx.getOrElse(txn.txHash, ArraySeq.empty)
-      val gas                   = gasByTx.getOrElse(txn.txHash, ArraySeq.empty)
-      val (gasAmount, gasPrice) = gas.headOption.getOrElse((0, U256.Zero))
+      val ins                                      = insByTx.getOrElse(txn.txHash, ArraySeq.empty)
+      val ous                                      = ousByTx.getOrElse(txn.txHash, ArraySeq.empty)
+      val gas                                      = gasByTx.getOrElse(txn.txHash, ArraySeq.empty)
+      val (gasAmount, gasPrice, scriptExecutionOk) = gas.headOption.getOrElse((0, U256.Zero, false))
```

Comment:
I think by default it should be `true`, which means the transaction succedded at least. It will false only when the transaction failed.